packages/next/src/client/components/react-dev-overlay/_experimental/internal/components/errors/error-overlay-layout/error-overlay-layout.tsx

```diff
@@ -51,6 +52,7 @@ interface ErrorOverlayLayoutProps extends ErrorBaseProps {
   dialogResizerRef?: React.RefObject<HTMLDivElement | null>
 }
 
+// TODO: Clean up and improve composition.
```

Comment:
Maybe include the name `TODO(<id>)` and be specific with the TODO comment. When others saw it they better have some context to know how to take action